src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimCallbackHandler.java

```diff
@@ -264,8 +264,7 @@ private CallbackResponse setGenericResponseTypeFlag(CallbackParams callbackParam
                 && isRespondent1.equals(YES))
             || (caseData.getRespondent2ClaimResponseType() != null
                 && caseData.getRespondent2ClaimResponseType().equals(
-                RespondentResponseType.FULL_DEFENCE)
-                && isRespondent1.equals(NO))
+                RespondentResponseType.FULL_DEFENCE))
```

Comment:
The removal of this additional condition renders the use of isRespondent1 redundant. Please test all scenarios to make sure nothing is broken due to this and if it works, we should also remove lines 255-259 around the use of isRespondent1.

Reply:
Hey @bhssamray isRespondent1 is still used in line 264!
i've tested the scenarios, and it doesn't look like i've broken it.

Reply:
The `isRespondent1` flag is used in CCD for page hide and show conditions - so it is still required. 

From the looks of the code, it was being used to ensure that respondent 1 said Full defence or respondent 2 said full defence `isRespondent1.equals(NO)` I am trying to understand why this is specific to respondent 2.

Perhaps in the first check in this if() isRespondent1.equals(YES) is redundant? We only really care about how each of the defendants have  actually respondent to a case